code/tests/test_app.py

```diff
@@ -11,7 +11,11 @@ def test_returns_correct_config(self):
         response = app.test_client().get("/api/config")
 
         assert response.status_code == 200
-        assert response.json == {"azureSpeechKey": None, "azureSpeechRegion": None}
+        assert response.json == {
+            "azureSpeechKey": None,
+            "azureSpeechRegion": None,
+            "OPENAI_API_BASE": "https://.openai.azure.com/",
```

Comment:
It's probably correct - but what's the significance of the . at the start of the URI?

Reply:
It's because there's no `AZURE_OPENAI_RESOURCE` env var set for this test.

Reply:
I feel like it would be good to have some sort of config/yaml file that sets a baseline of env vars for all the tests, then each individual test could remove/reconfigure the env vars as appropriate, but that could probably be done later